### PR TITLE
[2057] - Fix views/display submenus in OCIODisplay app

### PR DESCRIPTION
**Pull request by hannahmkrasnick**
This fixes the bug where the "Views" submenu (renamed from "Transform") doesn't update when a new "Display" is chosen (renamed submenu from "Device"). 

https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/2057

### CODE REVIEW

File: src/apps/ociodisplay/main.cpp [hunk 1]
```diff
@@ -52,22 +51,21 @@ std::string g_inputColorSpace;
...
+                                                                                                 {"Good", OCIO::OPTIMIZATION_GOOD},
+                                                                                                 {"Draft", OCIO::OPTIMIZATION_DRAFT}}};
 
-float g_exposure_fstop{ 0.0f };
-float g_display_gamma{ 1.0f };
-int g_channelHot[4]{ 1, 1, 1, 1 };  // show rgb
+float g_exposure_fstop{0.0f};
+float g_display_gamma{1.0f};
+int g_channelHot[4]{1, 1, 1, 1}; // show rgb
+int viewsMenuID;
```
Comment (doug-walker) [inline src/apps/ociodisplay/main.cpp:65]: The other global variables have a "g_" prefix, please follow that style.

**Review by doug-walker — CHANGES_REQUESTED** [review]
Thanks for this contribution Hannah!

Apologies for the delayed review, things have been very busy. Just one minor request below.

Minor issue, but I'm noticing a large number of white space changes. Not sure if that was you or you have an auto-formatter turned on. Although the changes are not bad, we generally don't like people to reformat white space in areas of the code they are not modifying. It makes the PR more difficult to review and it makes the git blame history of the file harder to follow. I won't ask you to waste your time reverting the changes, but please keep it in mind for future contributions.

**Comment by doug-walker** [comment]
@hannahmkrasnick , we would like to merge this in time for the OCIO 2.4.1 release around Dec. 9. Have you been able to make progress on getting the CLA signed?

**Review by doug-walker — APPROVED** [review]
This is very helpful, Hannah!  Thanks for getting the CLA signed, we will include this in OCIO 2.4.1.

**Review by cozdas — APPROVED** [review]

**Pull request merged**